sdk/python/symbolchain/nem/TransactionFactory.py

```diff
@@ -26,12 +25,19 @@ def __init__(self, network, type_parsing_rules=None):
 			nem_type_converter,
 			extend_type_parsing_rules(nem_type_converter, type_parsing_rules))
 
+	def _create_and_extend(self, transaction_descriptor):
+		transaction = self._create(transaction_descriptor, nc.TransactionFactory)
+		self._auto_encode_strings(transaction_descriptor, transaction)
+
+		# hack: explicitely translate transfer message
```

Comment:
why hack?

Reply:
cause message in nem is actually a struct :|
(messageType, message), so message content is not visible on top-level and auto-encode only happens at top-level